app/models/appointment_summary.rb

```diff
@@ -4,6 +4,10 @@
 
 class AppointmentSummary < ActiveRecord::Base
   DIGITAL_BY_DEFAULT_START_DATE = Date.new(2016, 6, 21)
+  EMAIL_REGEXP = Regexp.union(
+    /\A[^\s][^@]+@([^@\.]+\.)+[^@\.]+[^\s]\z/,
+    /\A\z/
+  )
```

Comment:
Don't know how far we want to take this stuff but this allows in special characters too:

```ruby
[11] pry(main)> "t^^^@hiy$.com".match(EMAIL_REGEXP)
=> #<MatchData "t^^^@hiy$.com" 1:"hiy$.">
[13] pry(main)> "t£$%^&*()@test.com".match(EMAIL_REGEXP)
=> #<MatchData "t£$%^&*()@test.com" 1:"test.">
```